NORTRIP_multiroad_read_main_inputs.f90

```diff
@@ -73,35 +73,37 @@ subroutine NORTRIP_read_main_inputs
         open(unit_logfile,file=filename_log,status='old',position='append')
     endif
     
-    write(unit_logfile,'(A,4I5)') ' Start date: ', start_date_input(year_index),start_date_input(month_index),start_date_input(day_index),start_date_input(hour_index)
-    write(unit_logfile,'(A,4I5)') ' End date: ', end_date_input(year_index),end_date_input(month_index),end_date_input(day_index),end_date_input(hour_index)
+    write(unit_logfile,'(A,5I5)') ' Start date: ', start_date_input(year_index),start_date_input(month_index),start_date_input(day_index),start_date_input(hour_index),start_date_input(minute_index)
+    write(unit_logfile,'(A,5I5)') ' End date: ', end_date_input(year_index),end_date_input(month_index),end_date_input(day_index),end_date_input(hour_index),end_date_input(minute_index)
     
     !Calculate the number of hours between end and start dates
-    n_hours_input=int((date_to_number(end_date_input)-date_to_number(start_date_input))*24.+.5)+1
+    n_hours_input=int((date_to_number(end_date_input)-date_to_number(start_date_input))*24./timestep+.5)+1
     if (n_hours_input.lt.1) then
         !n_hours_input=n_hours_default
         write(unit_logfile,'(A)') ' ERROR: Number of hours is 0 or less. Stopping'
         STOP 5
     endif
-    write(unit_logfile,'(A,4I5)') ' Number of hours: ', n_hours_input
+    write(unit_logfile,'(A,4I5)') ' Number of timesteps: ', n_hours_input
```

Comment:
When the time step is different than 1 hour, the variable "n_hours_input" is treated as "number of time steps". Do we need one variable that is actually "number of hours", and another that is "number of timesteps"? Or is it ok to only have a "number of timesteps" variable. If so, the "n_hours_input" should be renamed?